src/components/Controls/Controls.test.tsx

```diff
@@ -17,37 +17,34 @@ describe('the Controls component', () => {
     expect((tester1 = true));
   });
 
-  // describe('when the user is active', () => {
-  //   mockIsUserActive.mockImplementation(() => true);
-  //
-  //   it('should have the "active" class', () => {
-  //     mockUseRoomState.mockImplementation(() => 'disconnected');
-  //     const wrapper = shallow(<Controls />);
-  //     expect(wrapper.find('div').prop('className')).toContain('showControls');
-  //   });
-  //
-  //   it('should not render the ToggleScreenShare and EndCall buttons when not connected to a room', () => {
-  //     mockUseRoomState.mockImplementation(() => 'disconnected');
-  //     const wrapper = shallow(<Controls />);
-  //     expect(wrapper.find('ToggleScreenShareButton').exists()).toBe(false);
```

Comment:
please put this line back in to make restoring the screen sharing feature easier.